src/CommunityToolkit.Maui/Views/WrapperControl.windows.cs

```diff
@@ -62,7 +61,7 @@ public void CleanUp()
 			Opacity = 1;
 		}
 
-		view.IsInNativeLayout = false;
+		//view.IsInNativeLayout = false;
```

Comment:
Can we delete this? I don't see an equivalent to the `IsInNativeLayout` property anymore 

```suggestion
```